common/views/components/Body/Body.js

```diff
@@ -35,19 +38,27 @@ const Map = dynamic(import('../Map/Map'), { ssr: false });
 
 type BodySlice = {|
   type: string,
-  weight: Weight,
+  weight?: Weight,
```

Comment:
We currently have several body slices that don't have a weight property. Making it optional was the less involved fix.